libs/@local/hash-validation/src/lib.rs

```diff
@@ -361,18 +366,18 @@ mod tests {
             .expect("entity type not resolved");
         let closed_entity_type = ClosedEntityType::from_resolve_data(entity_type, &resolved_data)
             .expect("Could not close entity type");
+        let closed_multi_entity_type =
+            ClosedMultiEntityType::from_multi_type_closed_schema(iter::once(closed_entity_type))
+                .expect("Could not close multi entity type");
 
         let entity_types = entity_types
             .into_iter()
             .map(|(entity_type_uuid, entity_type)| {
                 let resolved_data = ontology_type_resolver
                     .resolve_entity_type_metadata(entity_type_uuid)
                     .expect("entity type not resolved");
-                let entity_type_id = entity_type.id.clone();
-                let closed_entity_type =
-                    ClosedEntityType::from_resolve_data(entity_type, &resolved_data)
-                        .expect("Could not close church");
-                (entity_type_id, closed_entity_type)
+                ClosedEntityType::from_resolve_data(entity_type, &resolved_data)
+                    .expect("Could not close church")
```

Comment:
Test log in the wrong place?
```suggestion
                    .expect("Could not close entity type")
```